hazelcast/src/main/java/com/hazelcast/spi/impl/eventservice/impl/operations/AbstractRegistrationOperation.java

```diff
@@ -53,11 +52,6 @@ public final void run() throws Exception {
     protected abstract void runInternal() throws Exception;
 
     private void checkMemberListVersion() {
-        if (memberListVersion == -1) {
```

Comment:
This is already removed by https://github.com/hazelcast/hazelcast/commit/982fd3bcc1bf8bf528edfb84b463f841195c1408

Reply:
Rebased.